lib/topological_inventory/core/service_catalog_client.rb

```diff
@@ -0,0 +1,59 @@
+require "rest_client"
+require "topological_inventory/core/service_plan_client"
+
+module TopologicalInventory
+  module Core
+    class ServiceCatalogClient
+      def initialize(source)
+        @source = source
+      end
+
+      # TODO? Move this to a module in the API repo
+      def order_service_plan(plan_name, service_offering_name, additional_parameters)
+        payload = ServicePlanClient.new.build_payload(
```

Comment:
I don't think we need a new client type for this but not a big deal I'm good with it for now

Reply:
You're probably right now that the payload doesn't need to actually make the external calls. When I shifted it out it was going to have to make them and was going to be a much bigger pain to write a spec for than I thought it should've been. Moving it out alleviated that issue but yeah after we figured out it doesn't even need to make the external calls it could probably be moved back in if we want.